cmd/dbaas_user_show_redis.go

```diff
@@ -0,0 +1,70 @@
+package cmd
```

Comment:
Let's not add Redis.
Aiven is deprecating it, and it will be soon to be removed from our CLI as well.

Reply:
Removed in efdee4f2